sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -23,35 +25,56 @@ export type AddConfigurationSettingParam<T extends string | FeatureFlagValue | S
 export interface AddConfigurationSettingResponse extends ConfigurationSetting, SyncTokenHeaderField, HttpResponseField<SyncTokenHeaderField> {
 }
 
+// @public
+export type AppConfigurationApiVersion = (typeof AppConfigurationApiVersion)[keyof typeof AppConfigurationApiVersion];
+
+// @public
+export const AppConfigurationApiVersion: {
+    readonly Latest: "2022-11-01-preview";
+    readonly Stable: "1.0";
+    readonly "1.0": "1.0";
+    readonly "2022-11-01-preview": "2022-11-01-preview";
+};
+
```

Comment:
Let's set up some time to talk about this. I'm considering walking this design back in Form Recognizer, which is where it looks like this comes from. It has a lot of consequences that have to have very careful attention paid to them to avoid creating broken APIs.